src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java

```diff
@@ -87,6 +91,13 @@
     public StateFlow build() {
         return StateFlowBuilder.<FlowState.Main>flow(FLOW_NAME)
             .initial(DRAFT)
+                .transitionTo(ONE_RESPONDENT_REPRESENTATIVE).onlyIf(oneRespondentRepresentative)
```

Comment:
Do we need the interim states `ONE_RESPONDENT_REPRESENTATIVE` and `TWO_RESPONDENT_REPRESENTATIVES` at all? Can we not have 2 transitions to `CLAIM_SUBMITTED` setting the flags accordingly?

Reply:
Didn't think of doing that, seems a bit simpler. Will do it. :+1: